.github/workflows/fetcher.yml

```
@@ -7,27 +7,33 @@ on:
   workflow_dispatch:
 
 jobs:
+  gemfilesgetter: 
+    uses: stefanomunarini/relaton-data-rfcs/.github/workflows/gemfiles-getter.yml@main
   fetch:
+    needs: [gemfilesgetter]
     name: rfcsubseries rfc fetching
     runs-on: ubuntu-latest
     steps:
       - name: Checkout code
-        uses: actions/checkout@v2
-      - name: Get BibXML service’s Relaton version
-        run: echo "RELATON_VERSION=$(curl -L https://github.com/ietf-ribose/bibxml-service/raw/main/RELATON_VERSION.txt)" >> $GITHUB_ENV
-      - name: Test BibXML service’s Relaton version
-        run: echo $RELATON_VERSION
+        uses: actions/checkout@v3
       - name: Use Ruby
         uses: ruby/setup-ruby@v1
         with:
           ruby-version: 3.0
-      - name: Install gem
-        run: |
-          gem install relaton-cli  -v "~> $RELATON_VERSION"
+      - name: ls
+        run: ls
+      - name: bundle add relaton-cli
+        run: bundle remove relaton-cli & bundle add relaton-cli --github 'ietf-ribose/bibxml-service' --ref 'gemfilelock'
```

```suggestion
        run: |
          bundle remove relaton-cli
          bundle add relaton-cli --github 'ietf-ribose/bibxml-service' --ref 'gemfilelock'
```

To be on the same page, is any reasonable point to duplicate `relaton-cli` gem into `ietf-ribose/bibxml-service` and not use it from the original repository `relaton/relaton-cli`?

You can 'freeze' `relaton/relaton-cli` at any point which you need (releases, or event commits in git)